src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -401,11 +402,7 @@ public static long HostToNetworkOrder(long host)
 #if BIGENDIAN
```

Comment:
Nit: You can also replace this ifdef with `BitConverter.IsLittleEndian` check to make this code portable while you are on it.

`BitConverter.IsLittleEndian` is JIT intrinsic, so the native code should be identical.

Reply:
Fixed